app/scripts/modules/oracle/pipeline/stages/findAmi/findAmiStage.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+const angular = require('angular');
+
+import {
+  ACCOUNT_SERVICE,
+  PIPELINE_CONFIG_PROVIDER
+} from '@spinnaker/core';
+
+module.exports = angular.module('spinnaker.oraclebmcs.pipeline.stage.findAmiStage', [
+  PIPELINE_CONFIG_PROVIDER,
+  ACCOUNT_SERVICE,
+])
+  .config(function(pipelineConfigProvider) {
+    pipelineConfigProvider.registerStage({
+      provides: 'findImage',
+      cloudProvider: 'oraclebmcs',
+      templateUrl: require('./findAmiStage.html'),
+      executionDetailsUrl: require('./findAmiExecutionDetails.html'),
+      executionConfigSections: ['findImageConfig', 'taskStatus'],
+      validators: [
+        { type: 'requiredField', fieldName: 'cluster' },
+        { type: 'requiredField', fieldName: 'selectionStrategy', fieldLabel: 'Server Group Selection'},
+        { type: 'requiredField', fieldName: 'regions' },
+        { type: 'requiredField', fieldName: 'credentials' }
+      ]
+    });
+  }).controller('oraclebmcsFindAmiStageCtrl', ($scope, accountService) => {
+
+    const provider = 'oraclebmcs';
```

Comment:
This appears all over the place so should probably be factored out into a constants module or similar.